vizro-ai/examples/dashboard_ui/assets/custom.css

```diff
@@ -346,7 +333,8 @@ img#banner {
   flex-flow: column wrap;
   gap: 12px;
   justify-content: flex-start;
-  overflow: none;
+  /*Nadija, can you check what overflow behaviour you guys wanted here? none doesn't exist as a property*/
+  overflow: hidden;
```

Comment:
@maxschulz-COL @nadijagraca - are there any other vizro-ai demos where we have to change the CSS files? I could only find this assets folder in this repository. Can you both please double-check that the dashboard_ui looks as expected?

Because the file was previously called `custom_css.css` it wasn't detected as a stylesheet at least in my pycharm configuration, which is why it didn't flag wrong CSS properties before. After renaming the file, this was one of the properties flagged. `none` doesn't exist as a property - can you quickly check what behaviour you wanted here?

Reply:
@nadijagraca Could you take this one as you have done all the CSS for the `vizro-ai` 🙏

Reply:
I'll add the plugin in a separate PR - fortunately, we didn't have that many unmatched properties. I'll wait for @nadijagraca to return and double-check the vizro-ai demo 👍 There are some js deprecation warnings as well, so I'll take a look at that in the other PR: https://github.com/mckinsey/vizro/pull/897